assets/js/googlesitekit/components-gm2/Button/index.js

```diff
@@ -97,25 +92,24 @@ const Button = forwardRef(
 			return newTabText;
 		};
 
-		const ButtonComponent = (
+		const tooltipTitle = ! hideTooltipTitle
+			? title || customizedTooltip || ariaLabel
+			: null;
+
+		const buttonComponent = (
```

Comment:
Since this variable is not reused anymore, we should get rid of it and render the component inline. What do you think?